File: src/frontend/src/app/components/atendimento/lancamento/model/lancamento-financeiro-procedimento-dto.ts
import { Exclude, Expose } from 'class-transformer';

@Exclude()
export class LancamentoFinanceiroProcedimentoDTO {
  @Expose() id?: string;
  @Expose() procedimentoId?: string;
  @Expose() procedimentoCodigo?: string;
  @Expose() procedimentoDescricao?: string;
  @Expose() convenioId?: string;
  @Expose() convenioNome?: string;
  @Expose() tabelaItemId?: string;
  @Expose() valor?: number;
}

File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/atendimento/lancamento/dto/LancamentoFinanceiroProcedimentoDTO.java
package br.com.grupopipa.gestaointegrada.atendimento.lancamento.dto;

import java.math.BigDecimal;
import java.util.UUID;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class LancamentoFinanceiroProcedimentoDTO {

    private UUID id;
    private UUID procedimentoId;
    private String procedimentoCodigo;
    private String procedimentoDescricao;
    private UUID convenioId;
    private String convenioNome;
    private UUID tabelaItemId;
    private BigDecimal valor;
}
